server/statshouse/statshouse-client.cpp

```diff
@@ -8,20 +8,71 @@
 #include <unistd.h>
 
 #include "common/resolver.h"
+#include "common/server/stats.h"
+#include "common/stats/provider.h"
 #include "common/tl/constants/statshouse.h"
 #include "common/tl/methods/string.h"
 #include "net/net-connections.h"
 #include "runtime/critical_section.h"
 #include "server/server-log.h"
 
+class statshouse_stats_t : public stats_t {
```

Comment:
Let's put this class to anonymous namespace to emphasis that it`s private class for this cpp file